app/konnect/geo.md

```diff
@@ -19,6 +19,7 @@ Geographic regions allow you to also operate {{site.konnect_saas}} in a similar
 * EU
 * ME
 * US
+* IN (India)
```

Comment:
Curious if we should add the full names of all geos in brackets similar to India?